code/core/src/components/components/typography/elements/A.tsx

```diff
@@ -8,7 +8,10 @@ export const A = styled(Link)(withReset, ({ theme }) => ({
   lineHeight: '24px',
 
   color: theme.color.secondary,
-  textDecoration: 'none',
+  // Ensure WCAG Level A compliance (SC 1.4.1), see https://www.w3.org/WAI/WCAG22/Techniques/failures/F73
+  textDecoration: 'underline',
+  textDecorationThickness: '0.5px',
+  textUnderlineOffset: '0.11em',
```

Comment:
After testing in a browser, this feels a bit better:

```suggestion
    textUnderlineOffset: '0.2em',
```